scanpy/preprocessing/_simple.py

```diff
@@ -696,6 +696,12 @@ def _regress_out_chunk(data):
     from statsmodels.tools.sm_exceptions import PerfectSeparationError
 
     for col_index in range(data_chunk.shape[1]):
+
+        # if all values are identical, the statsmodel.api.GLM throws an error; but then no regression is necessary anyways...
```

Comment:
```suggestion
        # if all values are identical, the statsmodel.api.GLM throws an error;
        # but then no regression is necessary anyways...
```

Making it a bit easier to read